src/components/AppFooter/components/Footer.js

```diff
@@ -16,6 +16,14 @@ const Footer = styled.footer`
   min-height: ${spacingScale(4)};
   align-items: stretch;
   justify-content: space-between;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+  border-top: 1px solid ${({ theme }) => theme.COLOR_KEYLINE_SOLID};
+  padding-bottom: ${({ bannerEnabled }) =>
+    bannerEnabled ? spacingScale(6) : "0"};
```

Comment:
```suggestion
    bannerEnabled ? spacingScale(6) : 0};
```

Needs to be number 0 as opposed to string "0".